### PR TITLE
docs: quickstart review

### LINKED ISSUE #3968: docs: quick start review for v3.2
To review the existing documentation in the Quick start section and add the files  integration description

### PR DESCRIPTION
**Pull request by roksys**
* Closes #3968

### CODE REVIEW

File: docs/quickstart/crud-operations.rst [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 .. _crud-operations:
 
-Create, Display, Search Records
-===============================
+Create, Display, Upload File, Download File, Search Records
```
Comment (equadon) [inline docs/quickstart/crud-operations.rst]: Now the header is just repeating the sub-headers which seems unnecessary. What do you think?

File: docs/quickstart/crud-operations.rst [hunk 2]
```diff
@@ -85,13 +87,103 @@ fetch it via the REST API:
...
+Upload a file
+-------------
+
+You can upload a file to a record.
+
+.. code-block:: shell
+
+  # Create an example file
+  $ echo 'example' > example.txt
+  # Upload a file named example.txt to the record with pid of 1
```
Comment (topless) [inline docs/quickstart/crud-operations.rst]: Put an empty line between for readability

File: docs/quickstart/crud-operations.rst [hunk 3]
```diff
@@ -85,13 +87,103 @@ fetch it via the REST API:
...
+  "contents": [
+    {
+      "delete_marker": false,
+      "key": "example.txt",
+      "updated": "2019-11-19T10:26:32.780466",
+      "size": 8,
+      "created": "2019-11-19T10:26:32.769314",
+      "checksum": "md5:ddce269a1e3d054cae349621c198dd52",
+      "links": {
+        "self": "https://localhost:5000/api/records/%3Cinvenio_records_rest.utils.LazyPIDValue%20object%20at%200x1115ab400%3E/files/example.txt?bucket_id=6ed8acbd-72a3-47b7-b070-c7a22f6b6919",
```
Comment (topless) [inline docs/quickstart/crud-operations.rst]: Is `invenio_records_rest.utils.LazyPIDValue` supposed to be there?

File: docs/quickstart/crud-operations.rst [hunk 4]
```diff
@@ -85,13 +87,103 @@ fetch it via the REST API:
...
+-------------
+
+You can upload a file to a record.
+
+.. code-block:: shell
+
+  # Create an example file
+  $ echo 'example' > example.txt
+  # Upload a file named example.txt to the record with pid of 1
+  $ curl -k -X PUT http://localhost:5000/api/records/1/files/example.txt \
```
Comment (equadon) [inline docs/quickstart/crud-operations.rst]:
```suggestion
  $ curl -k -X PUT https://localhost:5000/api/records/1/files/example.txt \
```

File: docs/quickstart/crud-operations.rst (hunk 3 above)
Comment (roksys) [reply docs/quickstart/crud-operations.rst]: Nope, something was wrong with my Invenio instance.

**Comment by ntarocco** [comment]
Will take all comments into account.

**Pull request merged**